packages/shared/components/popups/MintNativeTokenConfirmationPopup.svelte

```diff
@@ -48,7 +47,7 @@
 
     async function prepareFoundryOutput(): Promise<void> {
         const outputData = buildFoundryOutputData(Number(totalSupply), Number(circulatingSupply), metadata, aliasId)
-        preparedOutput = await $selectedAccount.buildFoundryOutput(outputData)
+        const preparedOutput: IFoundryOutput = await $selectedAccount.buildFoundryOutput(outputData)
```

Comment:
Type can be inferred from the function return value
```suggestion
        const preparedOutput = await $selectedAccount.buildFoundryOutput(outputData)
```